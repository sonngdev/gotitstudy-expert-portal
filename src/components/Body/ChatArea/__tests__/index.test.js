import React from 'react';
import { shallow } from 'enzyme';
import { ChatArea } from '../index';

describe('<ChatArea />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ChatArea />);
  });

  it('renders', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });

  it('contains chat messages', () => {
    const messages = wrapper.find('ChatMessages');
    expect(messages.exists()).toBe(true);
  });

  it('contains a chat input field', () => {
    const input = wrapper.find('ChatInput');
    expect(input.exists()).toBe(true);
  });
});
