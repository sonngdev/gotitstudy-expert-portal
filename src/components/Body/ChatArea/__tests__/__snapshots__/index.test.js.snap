// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatArea /> renders 1`] = `
"<div>
  <ChatMessages />
  <ChatInput />
</div>"
`;
